sympy/series/fourier.py

```diff
@@ -170,6 +189,42 @@ def truncate(self, n=3):
 
         return Add(*terms)
 
+    def sigma_approximation(self, n=3):
+        """
+        Return sigma approximation of Fourier series with respect to order n.
+
+        Sigma approximation is a method allowing for reduction of Gibbs effect
+        in discontinuous function approximation with Fourier series partial sums.
+
```

Comment:
Yes, have a look at the screenshot.
![screenshot from 2016-11-16 23-10-08](https://cloud.githubusercontent.com/assets/23407847/20367959/b6158134-ac49-11e6-8bae-ca032fc347af.png)